packages/wrangler/src/__tests__/vectorize/vectorize.test.ts

```diff
@@ -0,0 +1,114 @@
+import { endEventLoop } from "../helpers/end-event-loop";
+import { mockConsoleMethods } from "../helpers/mock-console";
+import { runInTempDir } from "../helpers/run-in-tmp";
+import { runWrangler } from "../helpers/run-wrangler";
+
+describe("vectorize", () => {
```

Comment:
It would be great to see more tests here—currently it's just testing the help output, and there's no testing relating to `vectorize` functionality

Reply:
addressed.